src/storage/raw/store.rs

```diff
@@ -172,7 +172,7 @@ impl<'a, S: Snapshot> RawStoreInner<S> {
         let mut pairs = vec![];
         let mut row_count = 0;
         let mut time_slice_start = Instant::now();
-        while cursor.valid()? && pairs.len() < limit {
+        while cursor.valid()? {
```

Comment:
It seems that this modification would cause wrong result when `limit == 0`.

And this also causes the failure of unittest `test_txn_store_rawkv` (see the [log](https://ci.pingcap.net/blue/organizations/jenkins/tikv_ghpr_test/detail/tikv_ghpr_test/12889/pipeline/115/#step-326-log-202)).
To be specific, this [test](https://github.com/tikv/tikv/blob/1f3b5cfa804077add976144296bdf770cff0246f/tests/integrations/storage/test_storage.rs#L773):
```
store.raw_scan_ok("".to_string(), b"".to_vec(), None, 0, vec![]);
```

Reply:
Fixed.
Sorry, I forgot to run tests after my modification, I will pay attention next time.